app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -46,6 +53,15 @@ class HomeFragment : Fragment() {
         // todo: saved instance state?
         initTiles()
         initUrlInputView()
+
+        settingsButton.alpha = 0.4f
+        settingsButton.setOnFocusChangeListener { v, hasFocus ->
+            v.alpha = if (hasFocus) 1.0f else 0.4f
```

Comment:
fwiw, this can also be done with StateListDrawables (which can reference `#hex` or `@color/...`, since they're inflated into ColorDrawables, which contain alpha values).

In this case, I don't think it's worth it because of the added indirection/complexity/duplication of colors (since the color needs to be written twice, with different alpha values) but just wanted to keep you aware.